Improve README.md (#31)
eganjs/synth-a-py: README.md

![Build](https://github.com/eganjs/synth-a-py/workflows/ci/badge.svg)

Daring to tackle the problem of complex project configuration through code!

_synth-a-py_ is a framework to enable synthesising of project configuration using Python :snake:

> The framework provides the means to define a project with multiple modules, that when synthesised, creates files such as `pyproject.toml`, `.github/workflows/{ci,cd}.yml` and `makefile`, all in the right places with consistency.

Reviewer: ```suggestion
The framework provides the means to define a project with multiple modules, that when synthesised, creates files such as `pyproject.toml`, `.github/workflows/{ci,cd}.yml` and `makefile`.
```

Reviewer: I hope it's implied :)

Author: Given the choice, I'd prefer to be explicit :stuck_out_tongue:

> Although the project is written in Python, it is not limited to generating Python projects. Theoretically, it could be used for any problem that requires consistency between two or more files.

Reviewer: ```suggestion
This project could be used for any problem that requires consistency between two or more files.

Python example
```

Author: I've removed this part, I was never 100% comfortable with it.
